TLM/TLM/State/Configuration.cs

```diff
@@ -257,6 +287,13 @@ public class ExtCitizenData {
             }
         }
 
+        public const int CURRENT_VERSION = 1;
```

Comment:
Change to `2` due to me robbing v1 in PR #1463

Reply:
done
I suppose previous intention was to manage version using this:
```
        private const string DATA_ID = "TrafficManager_v1.0";
        private const string VERSION_INFO_DATA_ID = "TrafficManager_VersionInfo_v1.0";
```
Now we have inconsistent code.

Reply:
Yeah, that older string-based way of doing it was grim; your `int Version` approach is far easier to work with... beause it's actually usable lol.

Reply:
I guess if we moved to XML we need to also change the data ID.